images/linux/Ubuntu2004-README.md

```diff
@@ -4,31 +4,31 @@
 | [Clang/LLVM 10 will be set as a default one and Clang/LLVM 6 will be deprecated  for Ubuntu 20.04 on September, 23](https://github.com/actions/virtual-environments/issues/1536) |
 ***
 # Ubuntu 20.04.1 LTS
-- Image Version: 20200920.1
+- Image Version: 20201004.1
 
 ## Installed Software
 ### Language and Runtime
-- GNU C++ 7.5.0, 8.4.0, 9.3.0
+- GNU C++ 7.5.0, 8.4.0, 9.3.0, 10.0.1
 - GNU Fortran 8.4.0, 9.3.0
-- Clang 10.0.1, 8.0.1, 9.0.1
-- Erlang 11.0.3
+- Clang 8.0.1, 9.0.1, 10.0.0
```

Comment:
Why is it downgraded?

Reply:
https://github.com/actions/virtual-environments/pull/1667